### PR TITLE
Use last_commit.create_status directly

### PR DESCRIPTION
**Pull request by rnetser**

### CODE REVIEW

File: src/labels_by_user_input.py [hunk 1]
```diff
@@ -19,19 +23,39 @@ def labels_by_user_input(event_data, pull):
         pull=pull
     ):
         add_label(pull=pull, label=LABEL_VERIFIED)
-        set_commit_status_success_verify(commit=last_commit)
+        set_commit_status(
```
Comment (sarahbx) [inline src/labels_by_user_input.py:26]: Love the generic method, but any reason we shouldn't reduce this to:
```
last_commit.create_status(...)
```
?

Comment (myakove) [reply src/labels_by_user_input.py:26]: @sarahbx what do you suggest? To have last_commit class? or I missunderstood you?

Comment (sarahbx) [reply src/labels_by_user_input.py:26]: Sorry I was not clear.
`set_commit_status(commit=,...)` has no functionality other than as a wrapper for `commit.create_status(...)`
Do we require the wrapper?

Comment (myakove) [reply src/labels_by_user_input.py:26]: Well, you are right :) 
wrapper that do nothing is always unwanted.

Comment (rnetser) [reply src/labels_by_user_input.py]: done

**Review by sarahbx — APPROVED** [review]
lgtm

File: src/merge_status_label.py [hunk 2]
```diff
@@ -6,7 +7,7 @@ def merge_status_label(pull, commit):
         print("No commit given")
         return
 
-    label = "Ready for merge"
+    label = READY_FOR_MERGE
```
Comment (myakove) [inline src/merge_status_label.py:10]: Can be removed and used directly `label = READY_FOR_MERGE`

**Review by myakove — CHANGES_REQUESTED** [review]

File: src/merge_status_label.py (hunk 2 above)
Comment (rnetser) [reply src/merge_status_label.py]: done

**Comment by myakove** [comment]
Merging, but we have a problem
`Ready for merge` label was set to your `Test pr Ruty` PR instead of this one.
Seems like we take the wrong commit when the PRs are based on other PR 

@rnetser

**Pull request merged**